doc_src/Readme.md

```diff
@@ -106,44 +118,48 @@ The ones available in `doc_src` are only there for reference.
 
 ### macOS
 
-All the dependencies can be installed through MacPorts except for Saxon 6-5-5:
+`ant` can be installed with the Xcode tools:
+    `$ xcode-select --install`
+	
+All the other dependencies can be installed through [MacPorts](https://www.macports.org/) except for saxon and whc.
+    `$ sudo port install docbook-xsl-nons, docbook-xml-4.5 @5.0_1, fop @1.1_1, libxml2 @2.9.12_1`
+
+- Saxon 6-5-5 ("saxon")
+  https://sourceforge.net/projects/saxon/files/saxon6/
 
-- [MacPorts](https://www.macports.org/)
-- [Saxon 6-5-5](https://sourceforge.net/projects/saxon/files/saxon6/6.5.5/)
+- XMLmind Web Help Compiler ("whc")
+  https://www.xmlmind.com/ditac/whc.shtml
 
-Unzip the downloaded package and put it in your prefered location. The example
+Unzip the downloaded packages and put them in your prefered location. The example
 below uses the `/Applications` folder.
 
 If you use Macports, the `doc_src_paths.xml` settings are:
 
-    <property name="fop.home" value="/opt/local/share/java/fop/1.0" />
-    <property name="dbk" value="/opt/local/share/xsl/docbook-xsl" />
+<project>
+    <property name="fop.home" value="/opt/local/share/java/fop/1.1" />
+    <property name="dbk" value="/opt/local/share/xsl/docbook-xsl-nons" />
     <property name="saxon" value="/Applications/saxon6-5-5/saxon.jar" />
+	<property name="whc" value="/Application/whc-3_3_0/lib/whc.jar" />
+</project>
 
 and the `docbook-utf8.xsl` settings are:
 
-    <xsl:import href="file:///opt/local/share/xsl/docbook-xsl/html/docbook.xsl"/>
+    <xsl:import href="file:///opt/local/share/xsl/docbook-xsl-nons/html/docbook.xsl"/>
 
 The DocBook 4.5 DTD is located here:
 
-    /opt/local/var/macports/software/docbook-xml-4.5/4.5_0/opt/local/share/xml/docbook/4.5/
-
-Copy the `4.5` folder to the folder that is three folders "higher" than the
-DocBook documentation source files in your folder tree and rename it
-`/docbook-xml-4.5`.
-
-So, if you have your DocBook documentation source files in:
-
-    /path/to/omegat/branches/release-2-3/doc_src/hu/
-
-The `/docbook-xml-4.5` folder should be copied inside `branches`:
-
-    /path/to/omegat/branches/docbook-xml-4.5/
-
-Both `doc_src_paths.xml` and `docbook-utf8.xsl` must be copied at the same level
-as `docbook-xml-4.5`.
-
-The ones available in `doc_src` are only there for reference.
+    /opt/local/var/macports/software/docbook-xml-4.5/
+	
+- unzip the `docbook-xml-4.5-5.0_1.darwin_20.noarch.tbz2` package found
+inside.
+- rename the `4.5` folder found in `opt/local/share/xml/docbook/4.5/`
+  to `/docbook-xml-4.5`.
+- move that folder to the OmegaT source tree, to the folder "above" `doc_src`:
+    - /path/to/omegat/doc_src/en/
+    - /path/to/docbook-xml-4.5/
+- `doc_src_paths.xml` and `docbook-utf8.xsl` must be put at the same
+level as `docbook-xml-4.5`. The ones available in `doc_src` are only
+there for reference.
```

Comment:
1. Wouldn't it be easier to tell people to symlink to the installed files at `/opt/local/share/xml/docbook/4.5/`?
2. Sequential steps would probably be better as an ordered list an an unordered list